snow/consensus/snowball/nnary_snowflake.go

```diff
@@ -25,29 +24,20 @@ type nnarySnowflake struct {
 	// wrap the n-nary slush logic
 	nnarySlush
 
-	// betaVirtuous is the number of consecutive successful queries required for
-	// finalization on a virtuous instance.
-	betaVirtuous int
-
-	// betaRogue is the number of consecutive successful queries required for
-	// finalization on a rogue instance.
-	betaRogue int
+	// beta is the number of consecutive successful queries required for
+	// finalization.
+	beta int
 
 	// confidence tracks the number of successful polls in a row that have
 	// returned the preference
 	confidence int
 
-	// rogue tracks if this instance has multiple choices or only one
-	rogue bool
-
 	// finalized prevents the state from changing after the required number of
 	// consecutive polls has been reached
 	finalized bool
 }
 
-func (sf *nnarySnowflake) Add(choice ids.ID) {
-	sf.rogue = sf.rogue || choice != sf.preference
```

Comment:
q: what was happening here and why we dropped this?

Reply:
Previously, `BetaVirtuous` was the number of consecutive polls required to commit when we had only seen one choice (ie a virtuous proposal without a conflict) and `BetaRogue` was the number of consecutive polls required to commit when we had seen more than one choice.

This code differentiated between a virtuous and rogue decision, so we could decide which threshold was required to finalize a decision. Now that they're consolidated (required to be the same value before this change, so this is a no-op) we can clean up this code.